specification/keyvault/resource-manager/Microsoft.KeyVault/preview/2021-04-01-preview/examples/ManagedHsm_getPrivateEndpointConnection.json

```diff
@@ -0,0 +1,33 @@
+{
+  "parameters": {
+    "subscriptionId": "00000000-0000-0000-0000-000000000000",
+    "resourceGroupName": "sample-group",
+    "name": "sample-mhsm",
+    "privateEndpointConnectionName": "sample-pec",
+    "api-version": "2021-04-01-preview"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/sample-group/providers/Microsoft.KeyVault/managedhsms/sample-mhsm/privateEndpointConnections/sample-pec",
+        "name": "sample-pec",
+        "type": "Microsoft.KeyVault/managedhsms/privateEndpointConnections",
+        "etag": "",
```

Comment:
Could this be elided if null or empty? In some cases, it can lead to problems (especially if ever returned as an explicit `null`)

Reply:
No this will not be elided.